interpreter/interpreter_test.go

```diff
@@ -308,6 +308,10 @@ func TestBuiltinFunctions(t *testing.T) {
 		{`type enumerated E1 {red, green, blue}; var E1 testVar := blue; int2enum(0, "wrong")`, runtime.Errorf("second argument must be an enum value")},
 		{`type enumerated E1 {red, green, blue}; var E1 testVar := blue; int2enum(4, testVar)`, runtime.Errorf("id 4 does not exist in any ranges of Enum E1")},
 
+		{`type enumerated E1 {red, green, blue}; var E1 testVar := blue; enum2int(testVar);`, `2`},
+		{`type enumerated E1 {red, green, blue}; var E1 testVar := blue; enum2int("wrong")`, runtime.Errorf("Argument must be an enum value")},
+		{`type enumerated E1 {red(10..20), green, blue}; var E1 testVar := red; enum2int(testVar);`, `10`},
```

Comment:
Enumerated values in TTCN-3 do not only have a name, but also a value (in that sense they are more similar to Rust, than C++):
```suggestion
		{`type enumerated E1 {red(10..20), green, blue}; var E1 testVar := red(15); enum2int(testVar);`, `15`},
```